src/components/ContentSection.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+import EventCard from './EventCard'
+import { EventsListType } from '~/lib/sanity.interfaces'
+
+export default function ContentSection({ events }: EventsListType) {
+  console.log('events n component', events, typeof events)
+  //sort - > cards by date;
+  return (
+    <section className="py-16 ">
+      <h2 className="font-bold text-center text-[42px]  text-[#21272A] font-roboto ">
+        Event Title
```

Comment:
This text should be dynamic and come from schema as for example `mainTitle`
We probably need to change the schema a little bit 

```
export default {
  name: 'events',
  title: 'Events',
  type: 'document',
  fields: [
     {
      name: 'title',
      title: 'Main Title events',
      type: 'title',
     },
    {
      name: 'intro',
      title: 'Intro',
      type: 'title',
     },
    {
      name: 'items',
      type: 'array',
      title: 'Events Items',
      of: [
        {
          type: 'object',
          name: 'eventItem',
          fields: [
            {
              name: 'id',
              type: 'slug',
              title: 'ID',
            },
            {
              name: 'title',
              type: 'string',
              title: 'Title Item',
              validation: (Rule) => Rule.required(),
            },
           ... AND your schema
          ],
        },
      ],
}

```